geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/command/ExecuteFunction66.java

```diff
@@ -58,29 +61,45 @@
 
   private static final ExecuteFunction66 singleton = new ExecuteFunction66();
 
-  protected static volatile boolean ASYNC_TX_WARNING_ISSUED = false;
+  private static volatile boolean asyncTxWarningIssued;
 
-  static final ExecutorService execService =
+  private static final ExecutorService execService =
       LoggingExecutors.newCachedThreadPool("Function Execution Thread-", true);
 
   public static Command getCommand() {
     return singleton;
   }
 
-  ExecuteFunction66() {}
+  private final InternalFunctionExecutionService internalFunctionExecutionService;
+  private final ServerToClientFunctionResultSender65Factory serverToClientFunctionResultSender65Factory;
+  private final FunctionContextImplFactory functionContextImplFactory;
+
+  ExecuteFunction66() {
+    this(InternalFunctionService.getInternalFunctionExecutionService(),
+        new DefaultServerToClientFunctionResultSender65Factory(),
+        new DefaultFunctionContextImplFactory());
+  }
+
+  ExecuteFunction66(InternalFunctionExecutionService internalFunctionExecutionService,
```

Comment:
Not sure if this is another "test only" constructor, but if so, it is missing the annotation.  It is hard to tell from the diff in GitHub.